packages/lib/src/settings/options/layoutParams_navigationArrows_mouseCursorContainerMaxWidth.js

```diff
@@ -0,0 +1,17 @@
+import { INPUT_TYPES } from '../utils/constants';
+import { default as GALLERY_CONSTS } from '../../common/constants';
+
+export default {
+  title: 'Mouse Cursor Container Max Width',
+  isRelevant: (options) =>
+    options.scrollDirection === GALLERY_CONSTS.scrollDirection.HORIZONTAL &&
+    options.showArrows &&
+    options.arrowsPosition === GALLERY_CONSTS.arrowsPosition.MOUSE_CURSOR,
+  isRelevantDescription:
+    'Set a Horizontal gallery ("Scroll Direction" as "Horizontal"), set "Show Navigation Arrows" to "true" and set "Navigation Arrows Position" to "Mouse Cursor".',
+  type: INPUT_TYPES.NUMBER,
+  min: 0,
```

Comment:
min 0 wont make for a working feature. we can say 10. (its only for the slider of the playground or the other setting panels that could read this data as a config)